NuRadioMC/utilities/medium_base.py

```diff
@@ -420,6 +419,221 @@ def get_birefringence_index_of_refraction(self, position):
 
         return nx, ny, nz
 
+class IceModelExponentialPolynomial(IceModel):
+    """
+    Predefined ice model (to inherit from) with polynomial exponential shape of degree n
+    """
+    def __init__(self, a, z_0, z_shift=0 * units.meter, z_air_boundary=0 * units.meter, z_bottom=None,
+                 density_factor=0.8506 * (units.cm**3 / units.gram)):
+        """
+        Initiation of an exponential polynomial ice model.
+
+        .. math::
+
+            n(z) = \sum_{i=0}^{n} a_{i} \cdot \exp^{i}(z/z_0)
```

Comment:
can you actually write the full and correct equation here:
`n(z) = 1 + density_factor * sum_{i=0}^{n} a_i * exp(i * (z-z_shift) / z_0)`

Reply:
@boboeyen I fixed the equation. Please check if I understood everything correct.

Reply:
Yes, my bad. The original equation is not wrong, but then a_i represent the coefficients of the fitted density polynomial translated to ice which is confusing as the translation is done by the class itself.

Would it make more sense to give the function the polynomial coefficient of the ice translation directly, rather than making the class do the translation itself?